README.md

```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
 
 #### Posts
 
-| Property         | Type   | Description                                      |
-| ---------------- | ------ | ------------------------------------------------ |
-| objectId         | String | unique id for the user post (default field)      |
-| createdAt        | Date   | (default auto generated field)                   |
```

Comment:
Rather than having an generic name like object id, you can rename this column to postId

Reply:
Unfortunately through Parse it automatically creates objectId as a default field and I don't think I can change it.